entries/one.xml

```diff
@@ -59,6 +59,12 @@ $( "#foo" ).on( "click", function( event ) {
     </code></pre>
     <p>In other words, explicitly calling <code>.off()</code> from within a regularly-bound handler has exactly the same effect.</p>
     <p>If the first argument contains more than one space-separated event types, the event handler is called <em>once for each event type</em>.</p>
+    <pre><code>
+$( "#foo" ).one( "click mouseover", function() {
+  alert( "This could be displayed twice." );
```

Comment:
How about this?
```js
$( "#foo" ).one( "click mouseover", function( event ) {
  alert( "The first " + event.type + " event happened." );
});
```
That way it's clear it happens once for each event, although your text clarifications do that too.